### ISSUE TITLE
Is this still maintained?

### ISSUE THREAD

**Issue by uhlhosting** [koenbuyens/securityheaders#8]
If so any plans to add new Feature Policy headers support?

**Comment by iDuronto** [comment]
No, the dev ain't interested as you can see. So please fork and add stuff. :)

**Comment by JensTimmerman** [comment]
Has anyone taken up this challenge? Is there a well maintainged fork to be found somewhere? It would be nice if this could be pointed to here, otherwise I will have to get to this myself. (I wanted to remove Expect-ct (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Expect-CT )

Feature-policy has also changed to permissions policy (https://developer.mozilla.org/en-US/docs/Web/HTTP/Permissions_Policy )
